boards/x86/minnowboard/doc/minnowboard.rst

```diff
@@ -66,4 +79,19 @@ The following platform features are unsupported:
 
 
 
+Creating a GRUB2 Boot Loader Image from a Linux Host
+====================================================
+
+Follow the same steps documented for the :ref:`Galileo board <grub2>`.
+
+
+Booting Zephyr on the Minnowboard
+=================================
+
+The MinnowBoard by default will be running a 64bit firmware. To boot Zephyr, you
+will need to replace the 64bit firmware with the 32bit variant. Please follow
+the instructions for flashing and updating the firmware documented at the
+`MinnowBoard`_ website.
+
+
```

Comment:
This URL would get folks to the firmware update instructions:  https://minnowboard.org/tutorials/updating-the-firmware